### PR TITLE
fix(compile): support latest version of Bonita BPM (7.3.2)

### PR DESCRIPTION
**Pull request by amottier**
fix(compile): support latest version of Bonita BPM (7.3.2)

Previous version of documentation page was only compatible with version 7.0.0.
New version of the documentation page rely on an external script provided in a GitHub project that can be easily update for each release.

Closes [BBPMC-410](https://bonita.atlassian.net/browse/BBPMC-410), [BBPMC-407](https://bonita.atlassian.net/browse/BBPMC-407), [BBPMC-405](https://bonita.atlassian.net/browse/BBPMC-405)

### CODE REVIEW

**Comment by tnguyen1** [comment]
add to whitelist

File: md/building-community-edition-from-source.md [hunk 1]
```diff
@@ -1,158 +1,27 @@
 # Build Bonita BPM Community edition from source files
 
-This page explains how to build Bonita BPM Community edition from the source files.
+This page explains how to build Bonita BPM Community edition from the source.
```
Comment (redboul) [inline md/building-community-edition-from-source.md]: why is it different from the title?

File: md/building-community-edition-from-source.md [hunk 2]
```diff
@@ -1,158 +1,27 @@
...
 
-echo "# Building project bonita-tomcat-h2-listener / tag bonita-tomcat-h2-listener-1.0.1"
-git clone --branch bonita-tomcat-h2-listener-1.0.1 --single-branch git@github.com:bonitasoft/bonita-tomcat-h2-listener.git bonita-tomcat-h2-listener
-mvn clean install -DskipTests=true -f bonita-tomcat-h2-listener/pom.xml
+In order to build Bonita BPM you need to use the script provided in [Build Bonita BPM](https://github.com/Bonitasoft-Community/Build-Bonita-BPM) GitHub project. Use the branch of the project that match the version you want to build.
 
-echo "# Building project bonita-tomcat-valve / tag 7.0.55"
-git clone --branch 7.0.55 --single-branch git@github.com:bonitasoft/bonita-tomcat-valve.git bonita-tomcat-valve
-mvn clean install -DskipTests=true -f bonita-tomcat-valve/pom.xml
+Simply run `bash build-script-7.3.2.sh` and provide required information.
```
Comment (redboul) [inline md/building-community-edition-from-source.md]: does the version needs to be fixed? can we use something like `bash build-script-x.y.z.sh` ?

Comment (ghost) [reply md/building-community-edition-from-source.md]: From a version to another version of connectors might change so we need to provide a version of the script for each maintenance release.

File: md/building-community-edition-from-source.md (hunk 1 above)
Comment (ghost) [reply md/building-community-edition-from-source.md]: I didn't spot it in the title.

**Pull request closed without merging**